protoc-gen-openapiv2/internal/genopenapi/types_test.go

```diff
@@ -0,0 +1,108 @@
+package genopenapi
+
+import (
+	"encoding/json"
+	"strings"
+	"testing"
+
+	"gopkg.in/yaml.v2"
+)
+
+func newSpaceReplacer() *strings.Replacer {
+	return strings.NewReplacer(" ", "", "\n", "", "\t", "")
+}
+
+func TestRawExample(t *testing.T) {
+	t.Parallel()
+
+	testCases := [...]struct {
+		In  RawExample
+		Exp string
+	}{{
+		In:  RawExample(`1`),
+		Exp: `1`,
+	}, {
+		In:  RawExample(`"1"`),
+		Exp: `"1"`,
+	}, {
+		In: RawExample(`{"hello":"worldr"}`),
+		Exp: "hello:\n" +
+			"  worldr\n",
```

Comment:
What do you think about using a raw string for these literals? The indentation looks a bit off but it reads easier I think.
```suggestion
		Exp:
`hello:
  worldr
`,
```

Reply:
Yeah, it is nicer